modules/sandboxed-containers-deleting-kata-resources.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-kata-resources_{context}"]
+= Deleting {sandboxed-containers-first} Kata resources
+
+You can remove and uninstall the `kata` runtime and all its related resources, such as CRI-O config and `RuntimeClass`, from your cluster.
+
+.Prerequisites
+
+* You have {sandboxed-containers-first} 1.0.0 installed.
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Delete the `KataConfig` custom resource by running the following command:
++
+[source,terminal]
+----
+oc delete kataconfig <KataConfig_CR_Name>
+----
+
+The {sandboxed-containers-operator} removes all resources that were initially created to enable the runtime on your cluster. After you run the preceding command, your cluster is restored to the state that it was prior to the installation process.
```

Comment:
We could add a note here that it is now okay to delete the namespace. 


"You can now delete the `openshift-sandboxed-containers` namespace"

Reply:
@zanetworker I think it should be the `openshift-sandboxed-containers-operator` namespace right? Just based on the other comments by @cpmeadors, @c3d :)

Reply:
the correct namespace is openshift-sandboxed-containers-operator